src/components/ChatLoader/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './styles.scss'
+
+export const ChatLoader = ({ height, width, color, dotRadius, dotPositions, className }) => {
+  return (
+    <svg height={height} width={width} className={className}>
```

Comment:
All style guide component's classname should start with `sg ` and then add custom classes based on requirements.
```suggestion
    <svg height={height} width={width} className={`sg typing-loader ${className}`}>
```

Reply:
added sg to the class